js/ui/drop_down_editor/ui.drop_down_editor.d.ts

```diff
@@ -39,6 +43,14 @@ export interface dxDropDownEditorOptions<T = dxDropDownEditor> extends dxTextBox
      * @public
      */
     applyValueMode?: 'instantly' | 'useButtons';
+    /**
+     * @docid dxDropDownEditorOptions.dropDownOptions
+     * @type dxPopupOptions
+     * @default {}
+     * @prevFileNamespace DevExpress.ui
+     * @hidden
```

Comment:
Could we change it to
```suggestion
     * @public
```
and remove redefinition of the `dropDownOptions`?

Reply:
fixed